eZ/Publish/API/Repository/Values/Content/Query/Criterion/IsBookmarked.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Values\Content\Query\Criterion;
```

Comment:
All new classes should be in Ibexa namespace

Reply:
Fixed